.github/workflows/release-candidate.yaml

```diff
@@ -74,9 +81,18 @@ jobs:
 
   release-solidity-contracts:
     uses: ./.github/workflows/release-solidity-contracts.yaml
+  
+  release-typescript-sdk:
+    needs: [set-variables]
+    uses: ./.github/workflows/release-typescript-sdk.yaml
+    with:
+      sdk_version: ${{ needs.set-variables.outputs.tag }}
+      sdk_tag: "rc"
```

Comment:
Should we set the tag to include the version this is a release candidate for? As a tag it's not like `latest` or `stable`, it's a release candidate of a specific version. I think including the version would make it easier to avoid accidentally pulling some old `rc`